### PR TITLE
feat: banner routing to new docs site

### PR DESCRIPTION
**Pull request by nataliepina**
This repo is now for the archived docs site.
These changes are to add a notice that the site has moved.

Closes D2IQ-92031

# Description
We want to include a notice for users to route them to the new docs site - https://docs.d2iq.com. 
The text should read - "You are viewing the documentation for older DKP releases. If you're looking for the new help center that includes the latest DKP release, go to https://docs.d2iq.com/."

I've matched the warning banner to a similar style as the "NOTE:" messages throughout the site to keep a consistent UX for notices. I've included the info icon as another indicator of important information if the blue/purple color alone is not indicative enough, which it may not be for those with visual impairments such as color deficiencies.
The banner will display on the main landing page, and product landing pages for Konvoy, Kommander, and Kaptain (and not for old docs such as DC/OS and Dispatch except for their specific docs pages because there is one component handing that for all products). The banner will also be included throughout the specific docs pages, that way if users had an old link to a specific doc they will still be alerted to route to the new site. This should cover most of our bases. 

The trade-offs with these changes are a lot of style adjustments due to the rigid CSS architecture of the site currently. Adding the extra space for the banner at the top requires some of the main content to be bumped down so it's not cut off. This should be fine as it may be one of the final changes to this repository. I tested these changes on different browsers and different screen sizes.

## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-92031

### Preview

Screen Shot 2022-09-05 at 1 37 54 PM
Screen Shot 2022-09-05 at 1 38 37 PM
Screen Shot 2022-09-05 at 1 36 50 PM

You can preview the docs build using the following URL, adding the PR # where specified:

http://docs-d2iq-com-pr-4623.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mstruebing — APPROVED** [review]

**Comment by vacas5** [comment]
can we make sure to get docs approval on this implementation before merging?

**Review by vacas5 — APPROVED** [review]
amazing, great work

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
Looks great!!

**Pull request merged**
